### ISSUE TITLE
Steven Universe - Unleash the Light - CUSA25214

### ISSUE THREAD

**Issue by ArbestRi** [red-prig/fpps4-game-compatibility#657]
# Game name
Steven Universe - Unleash the Light

## Code
CUSA25214

## Used emulator's version
red-prig/fpPS4@f117332

## Current status
Boots

## Error
sceSysmoduleLoadModule:SCE_SYSMODULE_AV_PLAYER
## Description
Game boots with black screen but after error spams "Forcing submitDone to avoid TRC R4089 breach" every few seconds

## Screenshots
None

**Comment by Kimiegg** [comment]
## Game version
1.00

## Used emulator's version
red-prig/fpPS4@184368f

## Current status
Game boots into black screen (FPS starts going up and down up to 59 fps after Case 1)

## Error
```
nop nid:libSceAvPlayer:A3D79646448BF8CE:sceAvPlayerInitEx
``` 
## Case 1 (tested with `libSceAvPlayer`)
```
nop nid:libSceVdecCore:CF07298C9ED65204:sceVdecCoreQueryComputeResourceInfo
```

## Case 2 (tested with `libSceVdecCore` + Case 1)
```
nop nid:libSceVideodec2:4670E26DC1823CAC:sceVideodec2QueryComputeMemoryInfo
```
## Case 3 (tested with `libSceVideodec2` + Case 2)
```
nop nid:libSceGnmDriver:DBDA0ABCA5F3119A:sceGnmMapComputeQueue
```

**Comment by ArbestRi** [comment]
@Kimiegg Alright, we should see something form this game once red-prig implements sceGnmMapComputeQueue!

**Comment by Kimiegg** [comment]
> @Kimiegg Alright, we should see something form this game once red-prig implements sceGnmMapComputeQueue!

Yeah but don't expect it to be soon as that is a very big part of the emulator

**Comment by gandalfthewhite19890404** [comment]
Retest

**Comment by ArbestRi** [comment]
nop nid:libSceAvPlayer:C7CBAFB8538F6615:sceAvPlayerAddSourceEx

**Comment by gandalfthewhite19890404** [comment]
Retest

**Comment by Kimiegg** [comment]
## Game version
1.00

## Used emulator's version
red-prig/fpPS4@a1414b0

## Current status
Game boots into black screen (FPS starts going up and down up to 59 fps after Case 1)
## Errors
I don't think i am getting any errors, correct me if i am wrong
[log.txt](https://github.com/red-prig/fpps4-game-compatibility/files/10444819/log.txt)

**Comment by ArbestRi** [comment]
In latest build, game boots with black screen and with distorted audio.
